heureka/ui/src/components/services/ServicesDetails.jsx

```diff
@@ -3,127 +3,274 @@
+          userId: newOwner.id,
+        },
+        {
+          onSuccess: () => {
+            addMessage({
+              variant: "success",
+              text: `${newOwner.name} added as owner.`,
+            })
+            setSelectedOwners((prev) => [...prev, newOwner])
+            setShowComboBox(false)
+            setNewOwner(null)
+          },
+          onError: (error) => {
+            addMessage({
+              variant: "error",
+              text: parseError(error),
+            })
+          },
+        }
+      )
+    }
+  }
 
-                <DataGridCell>
-                  {componentInstance?.node?.componentVersion?.version}
-                </DataGridCell>
+  // Remove an owner from the service
+  const handleRemoveOwner = (userId) => {
+    removeOwnerMutation.mutate(
+      {
+        serviceId: service.id,
+        userId: userId,
+      },
+      {
+        onSuccess: () => {
+          const removedUser = selectedOwners.find(
+            (owner) => owner.id === userId
+          )
+          addMessage({
+            variant: "success",
+            text: `${removedUser.name} removed as owner.`,
+          })
+          setSelectedOwners((prev) =>
+            prev.filter((owner) => owner.id !== userId)
+          )
+          setConfirmRemoveOwner(null) // Close the modal
+        },
+        onError: (error) => {
+          addMessage({
+            variant: "error",
+            text: parseError(error),
+          })
+        },
+      }
+    )
+  }
 
-                <DataGridCell>
-                  {componentInstance?.node?.issueMatches?.totalCount}
-                </DataGridCell>
+  const handleCancelAdd = () => {
+    setShowComboBox(false)
+    setNewOwner(null)
+  }
 
-                <DataGridCell>
-                  {severityString(
-                    highestSeverity(
-                      componentInstance?.node?.issueMatches?.edges
+  return (
+    <Stack direction="vertical" gap="4">
+      <ContentHeading heading="Service Details" />
+
+      <DataGrid columns={2}>
+        <DataGridRow>
+          <DataGridHeadCell className="min-w-[150px]">Owners</DataGridHeadCell>
+          <DataGridCell>
+            {!showComboBox && (
+              <Stack gap="2" wrap={true} alignment="center">
+                {selectedOwners.length > 0 &&
+                  selectedOwners.map((owner, i) => (
+                    <Pill
+                      key={i}
+                      pillValue={owner?.name}
+                      pillValueLabel={owner?.name}
+                      closeable
+                      onClose={() => setConfirmRemoveOwner(owner?.id)}
+                    />
+                  ))}
+                <Icon
+                  color="jn-global-text"
+                  icon="addCircle"
+                  onClick={() => setShowComboBox(true)}
+                  className="ml-2"
+                />
+              </Stack>
+            )}
+
+            {showComboBox && (
```

Comment:
When the add user combobox is shown and the user clicks on another service in the services list then the details view is updated but the combobox is still shown. This is confusing. 
Either prevent list clicks when the combobox is shown or ensure that the "edit mode", i.e. combobox is visible is reset when the user clicks on another service in the list.